aten/src/ATen/native/TensorFactories.cpp

```diff
@@ -278,6 +279,34 @@ Tensor empty_names(
   return result;
 }
 
+Tensor empty_permuted_symint(SymIntArrayRef size, IntArrayRef physical_layout, c10::optional<ScalarType> dtype_opt,
```

Comment:
do you want any error checking that physical layout is a permutation of `range(len(physical_layout))`?

Reply:
Hmm, that's a good idea...